Sources/KSCrashInstallations/include/KSCrashInstallation.h

```diff
@@ -44,12 +44,30 @@ NS_SWIFT_NAME(CrashInstallation)
 @interface KSCrashInstallation : NSObject
 
 /** C Function to call during a crash report to give the callee an opportunity to
- * add to the report. NULL = ignore.
+ * add to the report. NULL = ignore (DEPRECATED).
+ *
+ * @deprecated Use `onCrashWithPolicy` for async-safety awareness (since v2.4.0).
+ * This callback does not receive policy information and may not handle crash
+ * scenarios safely.
  *
  * WARNING: Only call async-safe functions from this function! DO NOT call
  * Swift/Objective-C methods!!!
  */
-@property(atomic, readwrite, assign, nullable) KSReportWriteCallback onCrash;
+@property(atomic, readwrite, assign, nullable) KSReportWriteCallback onCrash
```

Comment:
We had a type mismatch. The typedef `KSReportWriteCallback` was declared outside `NS_ASSUME_NONNULL_BEGIN`, making the `writer` parameter nullable. But in `KSCrashConfiguration`, the same callback was declared as `void (^crashNotifyCallback)(const struct KSCrashReportWriter *writer)` inside `NS_ASSUME_NONNULL_BEGIN`, making the `writer` parameter non-null by default. We have to restore that API as it blows up the compilation.